docs/examples/perf_analyzer/README.md

```diff
@@ -0,0 +1,46 @@
+# perf_analyzer example with DALI image decoder
+
+
+## Run the example
+
+### tritonserver
+
+To run the tritonserver, first you need to serialize DALI pipeline and put it inside model repo.
+
+`setup_perf_analyzer_example.sh` is a convenience script, that automatizes setting up.
+Provided you have DALI wheel installed in your system, you can just call `sh ./setup_perf_analyzer_example.sh`
+
+When you have your model repo set up, you can run `tritonserver`. Be sure to fill `<path to model repo>` properly:
+
+    docker run -it --rm --shm-size=1g --ulimit memlock=-1 --gpus all --ulimit stack=67108864 -p8000:8000 -p8001:8001 -p8002:8002 --privileged -v <path to model repo>:/models nvcr.io/nvidia/tritonserver:XX.YY-py3 tritonserver --model-repository=/models
+
+### perf_analyzer
+
+1. Pick an image you like the most
+1. Run Triton's client docker container:
+ 
+    ```
+    docker run -it --net=host nvcr.io/nvidia/tritonserver:XX.YY-py3-clientsdk
+    ```
+
+1. Create a directory for the test image and put it there. 
+**IMPORTANT**: file must have the same name as the input (`DALI_INPUT_0` in our case)
+    
+    ```
+    mkdir test_image
+    touch test_image/DALI_INPUT_0  # it's not really "touch" - it's an image
```

Comment:
Why you need to `touch` it?

Reply:
As I wrote in the comment, it's not really `touch`, it's an image. I thought that since I create this directory a line above, it's be obvious. Is it unclear?

Reply:
So how about:
```
cp PATH_TO_YOUR_IMAGE test_image/DALI_INPUT_0
```

Reply:
Done